media/js/src/simulations/simulation4/whatAreNonlinearRegs.jsx

```diff
@@ -29,6 +29,24 @@ export const WhatAreNonLinearRegressions = ({
         }
     };
 
+    const getMysteryFormula = (key) => {
```

Comment:
This is nearly identical to the `getFormula()` function. Why not standardize the key names in the JSON, and feed `data` in as a parameter?

Speaking of changes to the database, would you like me to fix the decimal points to 2 for all of the data so we don't have to run the extra function?

Reply:
Yeah I'll refactor this later as I build it out more. I just want to get things on the page.